ChangelogGenerator/ChangelogGenerator/ChangelogGenerator.csproj

```diff
@@ -9,9 +9,10 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>ChangelogGenerator</RootNamespace>
     <AssemblyName>ChangelogGenerator</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
```

Comment:
Use SDK based proj + PackageReference?
😢

Reply:
Migrated. Hopefully I did that right?

Reply:
Yup, looks correct.